pages/dashboard/views/index.jsx

```diff
@@ -44,12 +50,136 @@ const Index = ({
+                        className="govuk-button"
+                        label={<Snippet>{`buttons.roleApply`}</Snippet>}
+                      /></p>
+                    }
+                  </Fragment>
 
-            <PanelList panels={establishments.map(establishment => <EstablishmentPanel key={establishment.id} { ...establishment } />)}/>
+                  <div className='separator' />
+
+                  <h3><Snippet>pil.title</Snippet></h3>
+                  <Fragment>
+                    {
+                      pil && pil.licenceNumber && (
+                        <dl>
+                          <dt><Snippet>personalLicenceNumber</Snippet></dt>
+                          <dd><Link page="pil.read" establishmentId={establishment.id} profileId={id} pilId={pil.id} label={pil.licenceNumber} /></dd>
+                        </dl>
+                      )
+                    }
+                    { !pil && <p className="add-margin"><Snippet>pil.noPil</Snippet></p> }
+                    { !pil && allowedActions[establishment.id].includes('pil.create') &&
+                        <p><Link
+                          page='pil.create' establishmentId={establishment.id} profileId={id}
+                          className="govuk-button"
+                          label={<Snippet>{`buttons.pilApply`}</Snippet>}
+                        /></p>
+                    }
+                  </Fragment>
+                  <div className='separator' />
+
+                  <h3><Snippet>contactDetails.title</Snippet></h3>
+                  {
+                    (address || telephone || email) && (
+                      <Fragment>
+                        <dl>
+                          {
+                            email && (
+                              <Fragment>
+                                <dt><Snippet>contactDetails.email</Snippet></dt>
+                                <dd><a href={`mailto:${email}`}>{email}</a></dd>
+                              </Fragment>
+                            )
+                          }
+                          {
+                            address && (
+                              <Fragment>
+                                <dt><Snippet>contactDetails.professionalAddress</Snippet></dt>
+                                <dd>{address}<br />{postcode}</dd>
+                              </Fragment>
+                            )
+                          }
+                          {
+                            telephone && (
+                              <Fragment>
+                                <dt><Snippet>contactDetails.telephone</Snippet></dt>
+                                <dd>{telephone}</dd>
+                              </Fragment>
+                            )
+                          }
+                        </dl>
+                      </Fragment>
+                    )
+                  }
+
+                </ExpandingPanel>);
+            })} />
```

Comment:
this looks _very_ similar to the profile page in asl-pages, could we import that?

Reply:
no, everything there is in expanding panels

Reply:
YTHO?

Reply:
we have the same page in 2 places, we should be asking why it's different - could we not add the expanding panels in profile page?